src/Node/Stream/StdIO.purs

```diff
@@ -0,0 +1,11 @@
+-- | The standard IO streams for the current process.
+module Node.Stream.StdIO where
```

Comment:
Does it make sense to put this in a `node-process` library like we have HTTP streams in `node-http`?

I'm wondering if process objects can be obtained in other ways than just `require('process')`.